index.js

```diff
@@ -36,6 +36,19 @@ function outcomeOutput(bool, inputWord) {
     document.querySelector('form').style.flexDirection = 'row-reverse'
 }
 
+function addTileRow() {
+    let rowId = "tileRow1"
+    let letterArray = guessedWord.split("")
+    letterArray.forEach((letter) => {
+        let div_tag = document.createElement('div')
```

Comment:
Could we use camel case in this function for consistency throughout the project?